### ISSUE TITLE
"bootstrap: timed out waiting for job event" on single-node cluster

### ISSUE THREAD

**Issue by temujin9** [flynn/flynn#2268]
debug-info: https://gist.github.com/9fdc38b9ce31cac35f59 

```
++ flynn-host bootstrap --peer-ips=10.100.39.120 --timeout 600
+ bootstrap_output='07:06:33.909808 check online-hosts
07:08:32.436278 require-env require-env
07:08:32.436303 resource-check resource-check
07:08:32.436879 run-app discoverd
07:09:55.266221 run-app discoverd error: bootstrap: timed out waiting for job event'
+ echo 'Bootstrap failed, collecting debugging info'
Bootstrap failed, collecting debugging info
+ flynn-host collect-debug-info
t=2015-12-09T07:09:55+0000 lvl=info msg="uploading logs and debug information to a private, anonymous gist"
t=2015-12-09T07:09:55+0000 lvl=info msg="this may take a while depending on the size of your logs"
t=2015-12-09T07:09:55+0000 lvl=info msg="getting flynn-host logs"
t=2015-12-09T07:09:55+0000 lvl=info msg="getting job logs"
t=2015-12-09T07:10:24+0000 lvl=eror msg="error getting job logs, falling back to on-disk logs" err="could not list hosts: Get http://127.0.0.1:1111/services/flynn-host/instances: dial tcp 127.0.0.1:1111: connection refused"
t=2015-12-09T07:10:24+0000 lvl=info msg="getting system information"
t=2015-12-09T07:10:24+0000 lvl=eror msg="error capturing output of \"bash -c tail -n 1000 /var/log/flynn/*.log\"" err="exit status 1"
t=2015-12-09T07:10:24+0000 lvl=info msg="creating anonymous gist"
t=2015-12-09T07:10:25+0000 lvl=info msg="debug information uploaded to: https://gist.github.com/9fdc38b9ce31cac35f59"
+ exit 1
```

**Comment by temujin9** [comment]
I'll keep the node up for a little while, in case you need more info harvested.

**Comment by titanous** [comment]
Is there anything in `/var/log/flynn`? If so, try running `flynn-host collect-debug-info` as root.

**Comment by temujin9** [comment]
So the first one was run as root . . . don't know why it missed this, save maybe that it was run too soon?

Here's a new one: https://gist.github.com/6559e76cbed37fa79b82

**Comment by titanous** [comment]
This was fixed by #2512.
